Add IPv6 adoption documentation (#773)
openstack-k8s-operators/data-plane-adoption: docs_user/modules/proc_adopting-image-service-with-block-storage-backend.adoc

. Create a new file, for example `glance_cinder.patch`, and include the following content:
+

> [NOTE]
> If you are using IPv6, remember to change the load balancer IP to one that is correct in your environment. +
> E.g: `metallb.universe.tf/loadBalancerIPs:  fd00:bbbb::80`

Reviewer: ```suggestion
[NOTE]
If you use IPv6, change the load balancer IP to the IP in your environment, for example,
 `metallb.universe.tf/loadBalancerIPs:  fd00:bbbb::80`.
```

> If you use IPv6, remember to change the load balancer IP in the yaml below to the load balancer IP from your environment, for example,
> `metallb.universe.tf/loadBalancerIPs:  fd00:bbbb::80`.

Reviewer: ```suggestion
If you use IPv6, change the load balancer IP to the load balancer IP from your environment, for example,
`metallb.universe.tf/loadBalancerIPs:  fd00:bbbb::80`.
```
@elvgarrui A lot of these notes that you added would be better placed after the code as a bullet point. I suggest that you merge this PR (when it's ready), and then I can create a separate doc PR and make those changes.
